css/style.css

```
@@ -15,5 +18,247 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.karma_colone{
+  font-size: 70px;
+}
+hr{
+  width: 95%;
+}
+
+.nav_ul a:hover {
+  background-color: rgb(244, 29, 29);
+}
+
+.main_back_img{
+  display: block;
+  
+  height: 300px;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  margin: 5px;
+  padding: 100px;
+  margin-left: 50px;
+  border-radius: 10px;
+  
+  
+}
+
+.three_imgs{
+  
+  height: 300px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin:auto;
+  width: 90%;
+  margin-bottom: 200px;
+  
+}
+
+.img_1{
+  height: 200px;
+  width: 200px;
+  margin: auto;
+  
+}
+
+.Every_needs{
```

The rest of your CSS class names are nice and consistent, it'd be great for this one to match as well :+1: 
```suggestion
.every_needs{
```